llvm/test/CodeGen/RISCV/rvv/sink-splat-operands.ll

```diff
@@ -2,7 +2,7 @@
 ; RUN: llc < %s -mtriple=riscv64 -mattr=+m,+v,+f -target-abi=lp64f \
 ; RUN:     | FileCheck %s
 
-define void @sink_splat_mul(ptr nocapture %a, i32 signext %x) {
+define void @sink_splat_mul(ptr %a, i32 %x) {
```

Comment:
Do we need to change these existing tests?

Reply:
It seems that this information is unnecessary, so I deleted it.

Reply:
Can you split that out into a separate NFC PR just so we can make sure it isn't affecting this diff?

Reply:
OK, fixed

Reply:
Thanks. Could you also separate out the new tests in this PR into a separate commit so we can see the diff? Feel free to force push if you need

Reply:
done